src/app/core/services/filters.service.ts

```diff
@@ -365,4 +378,92 @@ export class FiltersService {
     const milestones = this.milestoneService.milestones;
     return [...milestones, Milestone.PRWithoutMilestone, Milestone.IssueWithoutMilestone];
   }
+
+  private getGhFilterDeselectedLabels(deselectedLabels: Set<string>): string {
+    return Array.from(deselectedLabels)
+      .map((label) => BooleanConjunctions.EXCLUDE + FilterOptions.label + `\"${label}\"`)
+      .join(BooleanConjunctions.AND);
+  }
+
+  private getGhFilterLabels(labels: string[]): string {
+    return labels.map((label) => FilterOptions.label + `\"${label}\"`).join(BooleanConjunctions.AND);
+  }
+
+  private getGhFilterMilestones(milestones: string[]): string {
+    return milestones
+      .map((milestone) => (MilestoneFilter.hasOwnProperty(milestone) ? MilestoneFilter[milestone] : FilterOptions.milestone + milestone))
```

Comment:
Milestones also need to be enclosed with quotations similar to labels.

To reproduce: access WATcher repo using WATcher, select only "Release 1.1" under "Milestones" and press view on github.

<img width="473" alt="image" src="https://github.com/user-attachments/assets/8ddbedb3-48f7-4cc7-82a9-464b3a6f42ee" />

This can be fixed by wrapping them in quotations.

<img width="427" alt="image" src="https://github.com/user-attachments/assets/98150891-b58c-4546-b777-956122b0089d" />